VM/rel2vm_params.py

```diff
@@ -745,6 +746,11 @@ def optimise_vm_params(
         )
         success = False
 
+    bounds_invalid = validate_vm_bounds([c1, c2, c3, c4], srf_meta["corners"].tolist())
```

Comment:
We should probably print / log debug information if it fails based on this check. Rather than having to crank out the debugger just to figure out why it didn't go.